libyul/optimiser/SSATransform.cpp

```diff
@@ -207,20 +207,20 @@ void IntroduceControlFlowSSA::operator()(Switch& _switch)
 {
 	yulAssert(m_variablesToReassign.empty(), "");
 
-	std::set<YulString> toReassign;
+	std::vector<YulString> toReassign;
 	for (auto& c: _switch.cases)
 	{
 		(*this)(c.body);
-		toReassign += m_variablesToReassign;
+		util::concatenateVectorWithoutDuplicates(toReassign, m_variablesToReassign);
 	}
 
-	m_variablesToReassign += toReassign;
+	util::concatenateVectorWithoutDuplicates(m_variablesToReassign, toReassign);
 }
 
 void IntroduceControlFlowSSA::operator()(Block& _block)
 {
-	std::set<YulString> variablesDeclaredHere;
-	std::set<YulString> assignedVariables;
+	std::vector<YulString> variablesDeclaredHere;
+	std::vector<YulString> assignedVariables;
 
 	util::iterateReplacing(
 		_block.statements,
```

Comment:
Since you're modifying the code around here anyway, could you fix the indent level for `toPrepend` declaration below? The way it is now it looks confusing, as if it was not meant to be a part of the block or something.

Reply:
Yes boss 🫡